features/redmine.feature

```diff
@@ -0,0 +1,58 @@
+Feature: Redmine project tests
```

Comment:
Yep, thought about it but was confused the `Registration positive` scenario will look like
```ruby
  Scenario: Registration positive
    Then New user is registered
```

But anyways it's more clear now, fixed =)